pkg/httputil/logging.go

```diff
@@ -13,6 +14,7 @@ type contextKey string
 
 const (
 	RequestIDContextKey contextKey = "request_id"
+	AuditLogEndMessage  string     = "HTTP call ended"
```

Comment:
1. No need the type - constant will get the type while using it.
2. You can update the message to include `API call information` or something like that if it make more sense.
```suggestion
	AuditLogEndMessage     = "HTTP call ended"
```

Reply:
@nopcoder this change leads to [this](https://github.com/dominikh/go-tools/issues/866)